containers/nestjs/src/game/pong.ts

```diff
@@ -234,10 +234,11 @@ class Ball extends Rect {
       if (this.left <= paddle.left || this.left <= paddle.right) {
         this._pos.x = this.left <= paddle.left ? paddle.left - this._size.w : paddle.right
         const paddleCenter = paddle.top + paddle._size.h / 2
-        const ballCenter = this.top + this._size.h / 2
+        const ballCenter = this.top + this._size.h / 2 // TODO: This seems off, as it doesn't add half the width -Sander
         const distance = paddleCenter - ballCenter
         const distanceMultiplier = distance / (paddle._size.h / 2)
 
+        // TODO: Can't this just be `this.bottom > paddle.top && this.top < paddle.bottom`?
```

Comment:
I've did a lot of trial and error and this worked, so be my guest to fiddle with it, but I don't want to touch this anylonger hahaha